public/canonical.json

```diff
@@ -0,0 +1 @@
Quantity of exposed clusters for a CVE"]},{"solr_command":"index","content_type":"documentation","id":"web-rca-service","uri":"https://developers.redhat.com/api-catalog/api/web-rca-service","name":"Web-RCA Service | API Catalog and Documentation","description":"Web-RCA Service API","content_summaries":["Returns a list of errors","Get an error by id","Return a list of Incidents","Create a new Incident","Delete an Incident","Get an Incident by id","Update an Incident","Get Events in given Incident by Incident id","Create a new Event under the Incident","Get FollowUps in given Incident by Incident id","Get Notifications in given Incident by Incident id","Create a new Notification under the Incident","Get IncidentTaskRuns in given Incident by Incident id","Get Attachments in given Event by Event id","Create a new Attachment under the Event","Delete an attachment","Return an attachment by incident id, event id, and attachment id","Update an Attachment","Delete an event","Return an event by Incident id and Event id","Upate an Event","Delete a Follow up","Return a FollowUp by Incident id and FollowUp id","Return temporary auth token to open WebSocket","Delete a Notification","Return an Notification by Incident id and Notification id","Update a Notification","Return an IncidentTaskRun by Incident id and IncidentTaskRun id","Execute a report by report ID","Return temporary state token to compose Add to Slack link","Return the list of all users in the system","Get an User by id","Marry a slack id to a user id","Delete a slack user mapping"]},{"solr_command":"index","content_type":"documentation","id":"case-management","uri":"https://developers.redhat.com/api-catalog/api/case-management","name":"Case Management API | API Catalog and Documentation","description":"Support Services Case Management API","content_summaries":["Get all attachment metadata","Upload a case attachment","Upload request status of all attachments for a given case number","Delete attachment metadata","Check case attachment metadata exist or not","Upload request status","Search cases based on Search Criteria","Search based on Search Criteria","Fetches Recommendation based on Search Criteria","Create Recommendations","Search cases based on Search Criteria","Search based on Search Criteria","Find account by calling User","List Customer Accounts for Partner","List Partner Accounts for Customer","Find account by number","Retrieve list of users for given case group","Find contacts for an account","Get Business Hours","Create new case","Get Cases by filters","Get Case By Case Number","Update case","Get Comments on case","Create Comment on case","Get case comment by comment id","List the externalUpdates on the specified case","Add notified users to case","Delete notified user from case","List Sftp Attachments","Get status for SFTP Attachment uploaded by user","Update token for SFTP User","Reset token for SFTP User","Create upload only SFTP Token for one time use","Create token for SFTP User"]}]}
```

Comment:
It's really good to be able to see this output once, but lets delete and ignore this file. 
Like the sitemap.xml is a build artifact.

Reply:
done!